src/lint/collect-algorithm-diagnostics.ts

```diff
@@ -35,6 +35,17 @@ export function collectAlgorithmDiagnostics(
     let element = algorithm.element;
     let location = getLocation(dom, element);
 
+    if (location.endTag == null) {
+      lintingErrors.push({
+        ruleId: 'no-close-tag',
```

Comment:
`missing-close-tag` might be less ambiguous. Rules prefixed with `no-` in the eslint world usually imply that it is restricting the use of the thing.

Reply:
Done.